lib/active_merchant/billing/gateways/ogone.rb

```diff
@@ -296,6 +293,36 @@ def add_d3d(post, options)
         add_pair post, 'PARAMPLUS',       options[:paramplus]       if options[:paramplus]
         add_pair post, 'COMPLUS',         options[:complus]         if options[:complus]
         add_pair post, 'LANGUAGE',        options[:language]        if options[:language]
+        if  options[:three_ds_2]
+          browser_info = options[:three_ds_2][:browser_info]
+          ecom_postal = options[:billing_address]
+          if browser_info
+            add_pair post, 'BROWSERACCEPTHEADER', browser_info[:accept_header]
+            add_pair post, 'BROWSERCOLORDEPTH', browser_info[:depth]
+
+            # for 3ds v2.1 to v2.2 add BROWSERJAVASCRIPTENABLED: This boolean indicates whether your customers have enabled JavaScript in their browsers when making a purchase.
+            # he following BROWSER<tag> parameters will remain mandatory
+            post['BROWSERJAVASCRIPTENABLED'] = browser_info[:javascript].to_s unless browser_info[:javascript].nil?
```

Comment:
due theres no Boolean class, it's not possible to use value.is_a?(Boolean) instead of that I'll use 

`def add_pair(post, key, value)
  post[key] = value unless value.nil?
end
`

Reply:
due theres no Boolean class I'll use 

def add_pair(post, key, value)
  post[key] = value unless value.nil?
end